applications/trilinos_application/custom_processes/trilinos_variational_distance_calculation_process.h

```diff
@@ -86,9 +86,11 @@ class TrilinosVariationalDistanceCalculationProcess
         ModelPart &base_model_part,
         LinearSolverPointerType plinear_solver,
         unsigned int max_iterations = 10,
-        Flags Options = NOT_CALCULATE_EXACT_DISTANCES_TO_PLANE)
-        : VariationalDistanceCalculationProcess<TDim, TSparseSpace, TDenseSpace, TLinearSolver>(base_model_part, max_iterations, Options),
-        mrComm(rComm)
+        Flags Options = NOT_CALCULATE_EXACT_DISTANCES_TO_PLANE,
+        std::string aux_part_name = "StandardAuxPartName" )
+        : VariationalDistanceCalculationProcess<TDim, TSparseSpace, TDenseSpace, TLinearSolver>(base_model_part, max_iterations, Options, aux_part_name ),
+        mrComm(rComm),
+        mAuxModelPart( aux_part_name )
```

Comment:
this should be called $mAuxModelPartName$ to tell that it is a string

Reply:
Ok, done.